broker/src/main/java/org/apache/rocketmq/broker/topic/TopicConfigManager.java

```diff
@@ -68,7 +68,6 @@ public TopicConfigManager(BrokerController brokerController) {
         {
             String topic = TopicValidator.RMQ_SYS_SELF_TEST_TOPIC;
             TopicConfig topicConfig = new TopicConfig(topic);
-            TopicValidator.addSystemTopic(topic);
```

Comment:
Removing this line could lead to misunderstanding. The person who adds the system topic next time maybe forget to add it to the system topic list in TopicValidator. I suggest adding a default logic to scan all system topics and add them to TopicValidator after initiating TopicConfigManager.